test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/math/BigIntegerTest.scala

```diff
@@ -163,4 +163,26 @@ class BigIntegerTest {
     assertTrue(Arrays.equals(eBytesSignum, expSignum.toByteArray))
     assertTrue(Arrays.equals(exp.toByteArray, expSignum.toByteArray))
   }
+
+  @Test def `should be correctly formatted as a decimal string`(): Unit = {
```

Comment:
I guess `test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/FormatterTest.scala` is the right place for this tests.

Am I wrong?

Reply:
Thanks for the suggestion @catap , I've moved them in 8558942b216aa5bbfac18361a20bd39faf8d44a7